superglue/run.py

```diff
@@ -115,6 +115,13 @@ def main(args):
 
     # Initialize Emmental
     config = parse_arg_to_config(args)
+    # HACK: handle None in model_path, proper way to handle this in the
```

Comment:
We should try to think if there's a way to do this logic in the bash script... otherwise, hack lgtm

Reply:
Will handle it in the next release of Emmental